int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -460,47 +479,49 @@ function edqSetProWebConfiguration() {
 		}
 	];
 	window.EdqConfig.PRO_WEB_LAYOUT= edqProWebAddressLayout;
-	if (edqCountryLineSelector != null)
+	if (edqCountryLineSelector != null) {
 		window.EdqConfig.PRO_WEB_COUNTRY= countryAlpha3(edqCountryLineSelector.value);
-	else
+	}
+	else {
 		window.EdqConfig.PRO_WEB_COUNTRY= countryAlpha3(vDefaultCountry);
+	}
 	window.EdqConfig.PRO_WEB_CALLBACK= "edqValidateAddressCallBack";
 	window.EdqConfig.PRO_WEB_MAPPING= [
 		{
 			field: edqAddressLine1Selector,
-			elements: ["Formatted Address 2"],
+			elements: ["address.addressLine1"],
 			modalFieldSelector: "#interaction-address--original-address-line-one",
 		},
 		{
 			field: edqAddressLine2Selector,
-			elements: ["AddressLine2"],
+			elements: ["address.addressLine2"],
 			transformation: function(el) { return "@" + (el.value || el.innerText) },
 			modalFieldSelector: "#interaction-address--original-address-line-two",
 		},
 		{
 			field: edqCityLineSelector,
-			elements: ["City name"],
+			elements: ["address.locality"],
 			transformation: function(el) { return "@" + (el.value || el.innerText) },
 			modalFieldSelector: "#interaction-address--original-locality",
 		},
 		{
 			field: edqStateLineSelector,
-			elements: ["State code"],
+			elements: ["address.province"],
 			modalFieldSelector: "#interaction-address--original-province",
 		},
 		{
 			field: edqPostalLineSelector,
-			separator: "-",
-			elements: ["ZIP Code", "+4 code"],
+			elements: ["address.postalCode"],
 			modalFieldSelector: "#interaction-address--original-postal-code",
 		},
-		{
-			field: edqCountryLineSelector,
-			elements: ['Two character ISO country code'],
-			modalFieldSelector: "#",
-		},
 	];
 }
+/**
+ * Will manage custom code from the client to verify the results from Pro Web Address Verification callback.
+ * @param {JSON} edqProWebMetaDataJSON
+ *
```

Comment:
By the way, `JSON` is not a Javascript type. In this situation you'd right `object`.